src/widgets/widgetLayer/ui/WidgetLayer.tsx

```diff
@@ -66,6 +67,11 @@ const WidgetLayer = () => {
         {/* FIXME: 임시로 item A 및 B 추가. 향후 위젯 추가시 삭제 요망*/}
         <div key="a">Item A</div>
         <div key="b">Item B</div>
+        {visibleState.MEMO.isVisible && (
+          <div key={WIDGET_KEYS.MEMO}>
+            <MemoContainer height="300" />
```

Comment:
```suggestion
            <MemoContainer height={getHeight(WIDGET_KEYS.MEMO)} />
```

지금 정확히 적용은 안되는데 widget에 hover했을 때, 닫힘버튼이 포함 된 설정바가 어디에 나오는지에 대한 height입니다.